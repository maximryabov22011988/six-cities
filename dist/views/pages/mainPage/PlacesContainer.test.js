"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const React = require("react");
const renderer = require("react-test-renderer");
const PlacesContainer_1 = require("./PlacesContainer");
it('PlacesContainer renders correctly with mandatory props', () => {
    const tree = renderer
        .create(React.createElement(PlacesContainer_1.default, { isEmpty: false, leftPanel: null, rightPanel: null })).toJSON();
    expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=PlacesContainer.test.js.map