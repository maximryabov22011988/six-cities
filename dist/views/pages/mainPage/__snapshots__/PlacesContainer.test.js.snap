// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlacesContainer renders correctly with mandatory props 1`] = `
<div
  className="cities__places-wrapper"
>
  <div
    className="cities__places-container container"
  >
    <div
      className="cities__right-section"
    >
      <section
        className="cities__map map"
      />
    </div>
  </div>
</div>
`;
